addons/sale_project/tests/test_project_profitability.py

```diff
@@ -141,12 +205,12 @@ def test_project_profitability(self):
                     ],
                     'total': {
                         'to_invoice': 0.0,
-                        'invoiced': self.delivery_service_order_line.untaxed_amount_invoiced,
+                        'invoiced': self.delivery_service_order_line.untaxed_amount_invoiced+100,
```

Comment:
You can let the `+` breathing a little bit. 😇 
```suggestion
                        'invoiced': self.delivery_service_order_line.untaxed_amount_invoiced + 100,
```